airflow/api/common/experimental/trigger_dag.py

```diff
@@ -122,7 +122,14 @@ def trigger_dag(
     dag_model = DagModel.get_current(dag_id)
     if dag_model is None:
         raise DagNotFound("Dag id {} not found in DagModel".format(dag_id))
-    dagbag = DagBag(dag_folder=dag_model.fileloc)
+    try:
+        store_serialized_dags = conf.getboolean('core', 'store_serialized_dags')
+    except Exception:
+        store_serialized_dags = False
```

Comment:
This should never throw an exception as the config is in the default_airflow.cfg

Reply:
This probably point to the fact that i should have written a test or e2e test this case but what will happen if core.store_serialized_dags is not specified KeyError or will it coerce to false?

Reply:
core.store_serialized_dags always exists because it is defined in the default configuration. https://github.com/apache/airflow/blob/master/airflow/config_templates/default_airflow.cfg
https://github.com/apache/airflow/blob/master/airflow/configuration.py#L174-L176

Reply:
Funny, without it all tests fail.
![image](https://user-images.githubusercontent.com/4841572/73459243-ccd5ff80-4376-11ea-8055-5c542e48218c.png)

Reply:
It was another conf. It was argument from POST body to DAG run.
For:
curl -X POST http://localhost:8080/api/experimental/dags/tutorial/dag_runs -H 'Cache-Control: no-cache' -H 'Content-Type: application/json' -d '{"conf":"{\"key\":\"value\"}"}'
It would have been:
{"key":"value"}
Unit test fails. My end to end test fails.
It is work in progress.

Reply:
Ok, fixed, works in tests and e2e